librz/core/cmd_descs.c

```diff
@@ -598,6 +599,20 @@ static const RzCmdDescHelp remote_rap_bg_help = {
 	.args = remote_rap_bg_args,
 };
 
+static const RzCmdDescArg cmd_help_search_args[] = {
+	{
+		.name = "search_cmd",
+		.type = RZ_CMD_ARG_TYPE_STRING,
+		.optional = true,
+
```

Comment:
was this created by the generator script? This empty line seems weird.

Reply:
yes.